src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -55,6 +71,20 @@ public String getDescription(Field field) {
                     return Localization.lang("Edition of a printed publication.");
                 case EDITOR:
                     return Localization.lang("Editor(s) of the work or the main publication, depending on the type of the entry.");
+                case EDITORA:
+                    return Localization.lang("A secondary editor performing a different editorial role, such as compiling, redacting, etc.");
+                case EDITORB:
+                    return Localization.lang("Another secondary editor performing a different role.");
+                case EDITORC:
+                    return Localization.lang("Another secondary editor performing a different role.");
+                case EDITORTYPE:
+                    return Localization.lang("The type of editorial role performed by the editor. Roles supported by default are \"editor\", \"compiler\", \"founder\", \"continuator\", \"redactor\", \"reviser\", \"collaborator\", \"organizer\". The role \"editor\" is the default. In this case, the field is omissible.");
```

Comment:
The different supported roles should be already clear as they are the options in the dropdown field.